bundles/automation/org.eclipse.smarthome.automation.commands/src/main/java/org/eclipse/smarthome/automation/internal/commands/CommandlineTemplateProvider.java

```diff
@@ -103,11 +103,14 @@ public void exportTemplates(String parserType, Set<RuleTemplate> set, File file)
      * @see AutomationCommandsPluggable#importTemplates(String, URL)
      */
     public Set<RuleTemplate> importTemplates(String parserType, URL url) throws IOException, ParsingException {
-        InputStreamReader inputStreamReader = null;
         Parser<RuleTemplate> parser = parsers.get(parserType);
-        if (parser == null) {
-            inputStreamReader = new InputStreamReader(new BufferedInputStream(url.openStream()));
-            return importData(url, parser, inputStreamReader);
+        if (parser != null) {
+            InputStreamReader inputStreamReader = new InputStreamReader(new BufferedInputStream(url.openStream()));
```

Comment:
I didn't know that. The Eclipse IDE shows JavaSE-1.7.
IIRC the EnOcean part uses also another compatibility.
Where can I see which parts of ESH using other requirements?